modules/apps/object/object-rest-test/src/testIntegration/java/com/liferay/object/rest/internal/manager/v1_0/test/DefaultObjectEntryManagerImplTest.java

```diff
@@ -1972,6 +1976,77 @@ public void testGetObjectEntryRelatedObjectEntriesWithAccountEntryRestricted()
 			objectRelationship2.getObjectRelationshipId());
 	}
 
+	@Test
+	public void testPartialUpdateObjectEntry() throws Exception {
+		ObjectEntry objectEntry = _defaultObjectEntryManager.addObjectEntry(
+			_dtoConverterContext, _objectDefinition2,
+			new ObjectEntry() {
+				{
+					properties = HashMapBuilder.<String, Object>put(
+						"dateObjectFieldName", "2023-08-10"
+					).put(
+						"decimalObjectFieldName", 2.7
+					).put(
+						"integerObjectFieldName", RandomTestUtil.randomInt()
```

Comment:
I researched about `RandomTestUtil.randomInt()` usage, and it is mostly used along with others random methods.
Also you are setting the others numbered values directly, I suggest you follow this pattern and set all numbers hardcoded.